packages/main/test/sap/ui/webcomponents/main/pages/kitchen-scripts.js

```diff
@@ -3,7 +3,7 @@ document.addEventListener("DOMContentLoaded", function(event) {
 	var mutated = false;
 	var sideNav = document.getElementById("side-nav");
 	var mainContent = document.getElementById("main-content");
-	var Configuration = sap.ui.getWCCore().getConfiguration();
+	var Configuration = window["sap-ui-webcomponents-main-bundle"].configuration;
```

Comment:
just to remind, that if you decide to change it above, don`t forget to rename it here as well.